packages/react-reconciler/src/ReactFiberScheduler.js

```diff
@@ -1094,8 +1094,14 @@ export default function<T, P, I, TI, HI, PI, C, CC, CX, PL>(
   }
 
   function computeInteractiveExpiration(currentTime: ExpirationTime) {
-    // Should complete within ~500ms. 600ms max.
-    const expirationMs = 500;
+    let expirationMs;
+    if (__DEV__) {
+      // Should complete within ~500ms. 600ms max.
```

Comment:
Would be good to document the rationale here. It's because relying on expiration is an indicator that you could've been doing better and you should see that opportunity during dev.

In prod, it's better to opt for slightly better UX if you fail.

Reply:
Thanks - will update the comment.